go.mod

```diff
@@ -55,21 +58,21 @@ require (
 	go.etcd.io/etcd/tests/v3 v3.5.9
 	go.nhat.io/aferocopy/v2 v2.0.1
 	go.opencensus.io v0.24.0
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.41.1
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.41.1
-	go.opentelemetry.io/contrib/propagators/b3 v1.16.1
-	go.opentelemetry.io/otel v1.15.1
-	go.opentelemetry.io/otel/bridge/opencensus v0.38.1
-	go.opentelemetry.io/otel/exporters/prometheus v0.38.1
-	go.opentelemetry.io/otel/metric v0.38.1
```

Comment:
Aktualizacia zavislosti, `metrics` v OpenTelemetry su uz stable (boli v RC, ked som to implementoval).